lib/server-utils.ts

```
@@ -0,0 +1,13 @@
+import 'server-only'
+import { dbType } from './constants';
+
+
+// Function to prepare ID parameter based on database type
+export const prepareIdParam = (id: string) => {
+  // For PostgreSQL, ensure the ID is treated as an integer
+  if (dbType === "postgres") {
+    return parseInt(id);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)